src/containers/ModelPanel/selectors.test.ts

```diff
@@ -0,0 +1,142 @@
+import {
+    getSelectAllVisibilityMap,
+    getSelectNoneVisibilityMap,
+    getIsSharedCheckboxIndeterminate,
+} from "./selectors";
+
+const mockUiDisplayData = [
+    {
+        title: "Macrophage",
+        key: "Macrophage",
+        children: [
+            {
+                label: "<unmodified>",
+                value: "",
+            },
+            {
+                label: "G0/G1 phase",
+                value: "G0/G1 phase",
+            },
+        ],
+    },
+    {
+        title: "Neutrophil",
+        key: "Neutrophil",
+        children: [
+            {
+                label: "<unmodified>",
+                value: "",
+            },
+            {
+                label: "G0/G1 phase",
+                value: "G0/G1 phase",
+            },
+        ],
+    },
+    {
+        title: "Dendritic cell",
+        key: "Dendritic cell",
```

Comment:
This is a preference thing, and I don't think we've been consistent with it either, but I kinda like naming test cases like this by a description instead of just test data names. Like, "agentWithNoStates" and "agentWithOneState"

Reply:
Ok, just updated the names.